uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt

```diff
@@ -63,13 +55,6 @@ private fun Input(
         Modifier
             .clickable(onClick = onClick)
             .fillMaxWidth()
-            .background(
-                if (isChecked) {
-                    ConferenceAppFeederTheme.filterMuskColor
-                } else {
-                    Color.Transparent
-                }
-            )
```

Comment:
Thank you for applying the changes!
Is there any reason to remove the background? 👀

Reply:
i thought this background musk is used to show which filter is applied.
and by this change, there are only ONE filter item and by checkbox, we can recognize whether favorite filter is selected or not.
so, i changed as above.
is it better to rollback about backgound ?

Reply:
I take a look at the material design guideline for a moment. :eyes:

Reply:
Certainly, it's not in the guidelines either. Let's erase it once. :+1:

Reply:
thank you for your check!!!